src/base_client.rs

```diff
@@ -328,6 +306,47 @@ impl Client {
         room.receive_state_event(event)
     }
 
+    /// Receive a presence event from an `IncomingResponse` and updates the client state.
```

Comment:
Let's calll this the sync response or something like that, Ruma calls `IncomingResponse` any response that a client might receive.